src/hooks/services/hooks.service.ts

```diff
@@ -342,6 +342,7 @@ export class HooksService {
       budgetInsightBank: {
         baseUrl: serviceAccountConfig?.baseUrl ?? config.budgetInsight.url,
         token: jsonWT.jwt_token,
+        clientId: serviceAccountConfig?.clientId,
```

Comment:
Should also work if there is no serviceAccountConfig, with default values
```suggestion
        clientId: serviceAccountConfig?.clientId ?? config.budgetInsight.,
```